app/models/manageiq/providers/microsoft/infra_manager/refresh_parser.rb

```diff
@@ -108,6 +112,11 @@ def parse_host(host)
       uid        = p[:ID]
       host_name  = p[:Name]
 
+      if p[:VirtualizationPlatform][:ToString].downcase.include?("vmwareesx")
+        $scvmm_log.debug("VMware ESX servers are not currently supported, skipping #{host_name}")
```

Comment:
Change the string `"vmwareesx"` to a constant array, something like `UNSUPPORTED_HOST_PLATFORMS = %w(vmwareesx)`.

And, change the log message to a warning.

Then, this block could be:
```ruby
host_platform = p[:VirtualizationPlatform][:ToString].downcase
if UNSUPPORTED_HOST_PLATFORMS.include? host_platform
  $scvmm_log.warn("#{host_platform} servers are not supported, skipping #{host_name}")
  return
end
```